env.example

```diff
@@ -4,6 +4,9 @@ NODE_ENV=development
 # PORT is the port used by the web server
 PORT=3000
 
+# AWS Public IPv4 address
+MY_PUBLIC_IP=(curl -s http://169.254.169.254/latest/meta-data/public-ipv4)
```

Comment:
Will this break on machines that don't have curl?

Reply:
The AWS EC2 instances that are running Ubuntu and Amazon Linux have curl preinstalled. I figured out how to get the ip address so these lines are no longer needed

Reply:
It won't let me comment on your third comment. But I made a script that will get the EC2 ip address and put it in the .env file so everything works now :)